plugins/scenario/worker/Worker.ts

```diff
@@ -60,10 +66,14 @@ export async function run<T, U, R>({ bases, config, worker }: WorkerData) {
   }
 
   for (const base of bases) {
-    const world = new World(hreForBase(base), base), dm = world.deploymentManager;
-    const delta = await dm.runDeployScript({ allMissing: true });
-    console.log(`[${base.name}] Deployed ${dm.counter} contracts, spent ${dm.spent} to initialize world 🗺`);
-    console.log(`[${base.name}]\n${dm.diffDelta(delta)}`);
+    const world = new World(base);
+    const dm = world.deploymentManager;
+    await runDeployScript(dm, base.name);
+
+    if (world.auxiliaryDeploymentManager) {
+      await world.auxiliaryDeploymentManager.spider();
```

Comment:
I think we might need to call `runDeployScript` on the auxiliary chain as well. Most of the time, it'll probably just be a no-op.

Reply:
Right, which should be a pretty rare case, but should be handled nonetheless

Reply:
If aux is meant for gov chain, I don't see why we'd ever need to deploy there?

Reply:
just encountered the need for this to be a redeploy. was adding something to the Goerli deploy.ts and needed the newly-added root to exist when running Mumbai scenarios.

I've added back in the runDeployScript for the auxiliary deployment

Reply:
I feel like this is more likely to lead to an error than prevent one, and in your case you should just run the deploy script changes for goerli so you get the address you need in roots/aliases for real